tests/legacy/simple_classif_training.py

```diff
@@ -26,7 +26,7 @@
 
 def main_train(dir_path, max_epochs: int = 20):
```

Comment:
@lantiga can you run this from the master without TB, only TBX?

Reply:
works for me, tested on CPU, CUDA, MPS